Codeherder19/bf-comp-challenge-1
# Feature/add images color and unique content

### What does this PR do?
- This PR adds unique content for this static comp including images for each Adventure Time character. Colors have been added, and there are two break points to create a responsive layout.

### What does it fix?
- Not a fix.

### Is this a feature or a fix?
- Feature.

### Where should the reviewer start?
- CD into bf-comp-challenge-1 and open index.html. Open dev tools and click on the responsive emulator to see how it looks on different screen sizes.

### How should this be tested?
- Open dev tools and click on the responsive emulator to see how it looks on different screen sizes.
